src/Settings.cpp

```diff
@@ -197,7 +197,11 @@ bool rpathFound(const string& rpath) { return rpath_to_fullpath.find(rpath) != r
 
 map<string, vector<string>> rpaths_per_file;
 vector<string> getRpathsForFile(const string& file) { return rpaths_per_file[file]; }
-void addRpathForFile(const string& file, const string& rpath) { rpaths_per_file[file].push_back(rpath); }
+void addRpathForFile(const string& file, const string& rpath) 
+{
+    if (std::find(rpaths_per_file[file].begin(), rpaths_per_file[file].end(), rpath) == rpaths_per_file[file].end()) return; 
```

Comment:
Shouldn't this be `!=`?

Reply:
Good point. It might be good to figure out why == fixed my issue, though.

Reply:
Hmm, looks like != also fixes the issue. /shrug